raw_data/preprocessed/africa/eswatini_english.txt

```diff
@@ -0,0 +1,291 @@
+
+Fixed Penalties
+(76) [1] The Minister may by regulation prescribe those offences against this Act in respect of which alleged offenders may pay a fine prescribed in the regulation instead of being tried by a court for the offence, provided that the maximum fine which may be prescribed for any one offence shall not exceed: [a] [100] Emalangeni if the alleged offender is required to pay the fine immediately in order to avoid prosecution; [b] [200] Emalangeni if the alleged offender is a natural person; or [c] [5000] Emalangeni if the alleged offender is legal person. [2] An inspector who has reason to believe that a person has committed an offence prescribed under subsection [1] may, at or about the time that the alleged offence is believed to have been committed, give an infringement notice to the alleged offender. [3] An infringement notice shall be in the prescribed form and shall in every case: [a] contain a description of the alleged offence; [b] state that if the alleged offender does not wish to be charged and tried by a court, the amount of money specified in the notice as being the modified penalty for the offence may be paid either: [i] to the authorised Public Officer identified in the notice within 28 days after the giving of the notice; or [ii] in the case of an offence in respect of which an on the spot fine is prescribed, to the Inspector . [4] An authorised Public Officer may, in a particular case, extend the period of 28 days for payment of the modified penalty and may do so whether or not that period has elapsed. [5] If the modified penalty specified in the infringement notice is paid within 28 days or such further time as is allowed and the notice is not withdrawn, the bringing of proceedings and the imposition of penalties are prevented to the same extent as they would be if the alleged offender had been convicted by a court of, and punished for, the alleged offences. [6] Any person who receives payment of a modified penalty, including an on the spot fine, shall give an official receipt for the money to the person who makes the payment. [7] The Director may, whether or not the modified penalty has been paid, withdraw the infringement notice by sending to the alleged offender a notice of withdrawal in the prescribed form. [8] If an infringement notice is withdrawn after payment of the modified penalty, the amount shall be refunded. [9] Payment of a modified penalty is not to be regarded as an admission for the purpose of any civil or criminal proceedings. [10] The Director may appoint any Public Officer other than an inspector to be an authorised person for the purposes of this section.
```

Comment:
Please remove the square brackets around the Emalangeni quantities. Also, please replace `5000` with `5,000`. Furthermore, please replace `is legal person` with `is a legal person`. Finally, please replace `Inspector .` with `Inspector.`.